openff/toolkit/utils/exceptions.py

```diff
@@ -51,7 +51,7 @@ class IncompatibleUnitError(OpenFFToolkitException):
     """
 
 
-class MissingDependencyError(OpenFFToolkitException):
+class MissingDependencyError(OpenFFToolkitException, ImportError):
```

Comment:
Is this diff supposed to be here?

Reply:
Yes, this is the change that changes the bug/undesired behavior. The other approach would be to catch this `MissingDependencyError` _instead of_ `ImportError` in `_ipython_display_` [here](https://github.com/openforcefield/openff-toolkit/blob/master/openff/toolkit/topology/molecule.py#L6436-L6439). I left this stand while #771 played out, and the discussion there seems to be in favor of this approach, so I left it in place.

Reply:
Thanks for the speedy review!